package.json

```diff
@@ -30,8 +32,9 @@
     "plugin"
   ],
   "scripts": {
-    "build": "rimraf dist && babel src --out-dir dist",
-    "clean": "npx rimraf node_modules dist build",
+    "clean-dist": "npx rimraf index.js utils.js gatsby-node.js",
```

Comment:
I was trying to reuse the files variable to delete output files ( something like $npm_package_files), but was unable to make it work. So I instead introduced a new step which is used in two places. Feel free to rename it to something more descriptive or use the variable if you know how.